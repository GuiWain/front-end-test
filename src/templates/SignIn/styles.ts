import styled, { css } from 'styled-components'
import UseFormSignIn from 'components/FormSignIn'
import UseButton from 'components/Button'

import UseBase from 'templates/Base'

import { Google } from '@styled-icons/evaicons-solid'

export const Base = styled(UseBase)``

export const Button = styled(UseButton)`
  width: 30%;
  margin: auto;
`
export const FormSignIn = styled(UseFormSignIn)``
export const GoogleIcon = styled(Google)``

export const SignIn = styled.section`
  ${({ theme }) => css`
    background-color: ${theme.colors.secondaryLightest};
    padding: 7rem 0;

    display: flex;
    flex-direction: column;
    gap: 2rem;
    justify-content: center;
    align-items: center;
  `}
`

export const Span = styled.span`
  ${({ theme }) => css`
    color: ${theme.colors.secondaryDark};
    font-size: ${theme.font.sizes.medium};
  `}
`
