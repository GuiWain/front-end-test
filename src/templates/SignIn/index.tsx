import * as S from './styles'

import { FormEvent, useState } from 'react'

import { FieldErrors, signInValidate } from 'utils/validations'
import { FormSignInValue } from 'components/FormSignIn'

const SignIn = () => {
  const [fieldError, setFieldError] = useState<FieldErrors>({})

  const handleSubmit = async (e: FormEvent, values: FormSignInValue) => {
    e.preventDefault()
    const errors = signInValidate(values)

    if (Object.keys(errors).length) {
      setFieldError(errors)
      return
    }

    setFieldError({})
  }

  return (
    <S.Base>
      <S.SignIn>
        <S.FormSignIn handleSubmit={handleSubmit} fieldError={fieldError} />
        <S.Span>Or</S.Span>
        <S.Button bg="transparent" icon={<S.GoogleIcon />}>
          Login by Google
        </S.Button>
      </S.SignIn>
    </S.Base>
  )
}

export default SignIn
