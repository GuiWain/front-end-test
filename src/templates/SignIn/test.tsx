import userEvent from '@testing-library/user-event'
import { fireEvent, render, screen } from 'utils/test-utils'

import SignIn from '.'

describe('<SignIn />', () => {
  it('should render the SignIn', () => {
    const { container } = render(<SignIn />)

    expect(container.firstChild).toMatchSnapshot()
  })

  it('must submit the form with success', () => {
    render(<SignIn />)

    const inputUsername = screen.getByPlaceholderText('Your email')
    const username = 'success@email.com'
    userEvent.type(inputUsername, username)

    const inputPassword = screen.getByLabelText(/Password/i)
    const password = 'success123456'
    userEvent.type(inputPassword, password)

    const submit = screen.getByText('Sign in')
    fireEvent.submit(submit)
  })

  it('must submit the form with invalid field', () => {
    render(<SignIn />)

    const inputUsername = screen.getByPlaceholderText('Your email')
    const username = 'error,email.com'
    userEvent.type(inputUsername, username)

    const inputPassword = screen.getByLabelText(/Password/i)
    const password = 'error'
    userEvent.type(inputPassword, password)

    const submit = screen.getByText('Sign in')
    fireEvent.submit(submit)
  })
})
