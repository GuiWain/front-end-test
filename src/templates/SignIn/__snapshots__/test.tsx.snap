// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignIn /> should render the SignIn 1`] = `
.c8 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c31 {
  cursor: pointer;
}

.c28 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #FFFFFF 0% 0% no-repeat padding-box;
  border: 0.1rem solid #CCC8C8;
  border-radius: 0.4rem;
  padding: 0 0.8rem;
  height: 3.6rem;
}

.c28:focus-within {
  box-shadow: 0rem 0.4rem 0.6rem #00000014;
  border: 0.1rem solid #3241A1;
}

.c29 {
  color: #4F4C4C;
  font: normal normal 400 1.4rem / 2.8rem 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  padding: 0.8rem 0;
  padding-left: 0.4rem;
  background: transparent;
  border: 0;
  outline: none;
  width: 100%;
}

.c29:-webkit-autofill {
  -webkit-box-shadow: 0 0 0 2.4 #EAEAEA inset;
  -webkit-filter: none;
  filter: none;
}

.c32 {
  color: #4F4C4C;
  font: normal normal 400 1.4rem / 2.8rem 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  padding: 0.8rem 0;
  padding-right: 0.4rem;
  background: transparent;
  border: 0;
  outline: none;
  width: calc(100% - 2.2rem);
}

.c32:-webkit-autofill {
  -webkit-box-shadow: 0 0 0 2.4 #EAEAEA inset;
  -webkit-filter: none;
  filter: none;
}

.c27 {
  font: normal normal 600 1.6rem / 4rem 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  -webkit-letter-spacing: -0.006rem;
  -moz-letter-spacing: -0.006rem;
  -ms-letter-spacing: -0.006rem;
  letter-spacing: -0.006rem;
  text-align: left;
  color: #4F4C4C;
  cursor: pointer;
}

.c30 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  color: #848080;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c30 > svg {
  width: 2rem;
  height: 100%;
}

.c18 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 0.6rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  border-radius: 0.4rem;
  -webkit-transition: all 0.5s ease-in-out;
  transition: all 0.5s ease-in-out;
  color: #FFFFFF;
  font-family: 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  font-weight: 700;
  font-size: 1.4rem;
  line-height: 2.6rem;
  height: 3.6rem;
  padding: 0.9rem 4.4rem;
  font-size: 1.4rem;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  background: #510161 0% 0% no-repeat padding-box;
  box-shadow: 0rem 0.4rem 0.8rem #00000014;
  border: none;
}

.c18:hover {
  background: #8802A3 0% 0% no-repeat padding-box;
  -webkit-transition: background 0.5s ease-in-out;
  transition: background 0.5s ease-in-out;
}

.c21 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 0.6rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  border-radius: 0.4rem;
  -webkit-transition: all 0.5s ease-in-out;
  transition: all 0.5s ease-in-out;
  color: #FFFFFF;
  font-family: 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  font-weight: 700;
  font-size: 1.4rem;
  line-height: 2.6rem;
  height: 3.6rem;
  padding: 0.9rem 4.4rem;
  font-size: 1.4rem;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  color: #2D2A2A;
  background: transparent 0% 0% no-repeat padding-box;
  border: 0.2rem solid #510161;
  box-shadow: 0rem 0.4rem 0.8rem #00000014;
}

.c21:hover {
  color: #FFFFFF;
  border-color: #8802A3;
  background: #8802A3 0% 0% no-repeat padding-box;
  -webkit-transition: background 0.5s ease-in-out;
  transition: background 0.5s ease-in-out;
}

.c33 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 0.6rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  border-radius: 0.4rem;
  -webkit-transition: all 0.5s ease-in-out;
  transition: all 0.5s ease-in-out;
  color: #FFFFFF;
  font-family: 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  font-weight: 700;
  font-size: 1.4rem;
  line-height: 2.6rem;
  width: 100%;
  height: 3.6rem;
  padding: 0.9rem 4.4rem;
  font-size: 1.4rem;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  background: #510161 0% 0% no-repeat padding-box;
  box-shadow: 0rem 0.4rem 0.8rem #00000014;
  border: none;
}

.c33:hover {
  background: #8802A3 0% 0% no-repeat padding-box;
  -webkit-transition: background 0.5s ease-in-out;
  transition: background 0.5s ease-in-out;
}

.c35 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 0.6rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  border-radius: 0.4rem;
  -webkit-transition: all 0.5s ease-in-out;
  transition: all 0.5s ease-in-out;
  color: #FFFFFF;
  font-family: 'Open Sans','Roboto','Helvetica Neue',sans-serif;
  font-weight: 700;
  font-size: 1.4rem;
  line-height: 2.6rem;
  height: 3.6rem;
  padding: 0.9rem 4.4rem;
  font-size: 1.4rem;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  color: #2D2A2A;
  background: transparent 0% 0% no-repeat padding-box;
  border: 0.2rem solid #510161;
  box-shadow: 0rem 0.4rem 0.8rem #00000014;
}

.c35 svg {
  width: 2rem;
}

.c35 svg + span {
  margin-left: 0.8rem;
}

.c35:hover {
  color: #FFFFFF;
  border-color: #8802A3;
  background: #8802A3 0% 0% no-repeat padding-box;
  -webkit-transition: background 0.5s ease-in-out;
  transition: background 0.5s ease-in-out;
}

.c26 {
  width: 35rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 1.5rem;
  background-color: #fff;
  box-shadow: 0px 0.3rem 0.6rem #00000029;
  padding: 3rem 4rem;
  margin: auto;
}

.c37 {
  background-color: #848080;
  width: 100%;
  left: 0;
  bottom: 0;
  padding: 1.5rem 0;
}

.c38 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 auto;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 1rem;
  margin: 0 auto 1.5rem;
}

.c39 {
  text-align: center;
  font-weight: 300;
  font-size: 2.4rem;
  color: #FFFFFF;
  margin: 0;
}

.c40 {
  width: 70%;
  display: block;
  margin: 0.5rem auto;
  text-align: center;
  font-weight: 300;
  font-size: 1.2rem;
  color: #FFFFFF;
}

.c7 {
  max-width: 100%;
  object-fit: contain;
}

.c3 {
  width: 8rem;
}

.c6 {
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c11 {
  width: 100%;
}

.c14 {
  list-style: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0 auto;
}

.c15 {
  font-weight: 700;
  font-size: 1.4rem;
  color: #848080;
  text-align: center;
  margin: 0 1rem;
  cursor: pointer;
  -webkit-transition: color 0.5s ease-in-out;
  transition: color 0.5s ease-in-out;
}

.c15:hover {
  color: #4F4C4C;
  -webkit-transition: color 0.5s ease-in-out;
  transition: color 0.5s ease-in-out;
}

.c1 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 7.5rem;
  background-color: #FFFFFF;
  z-index: 999;
  -webkit-transition: all 0.5s ease-in-out;
  transition: all 0.5s ease-in-out;
}

.c2 {
  width: 100%;
  height: 7.5rem;
  display: grid;
  grid-template-columns: 18% 60% 22%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  width: 7rem;
  margin-left: 2rem;
}

.c13 {
  justify-self: flex-end;
}

.c17 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  gap: 1rem;
}

.c20 {
  padding: 1rem 2rem;
  font-size: 1.4rem;
}

.c23 {
  padding: 1rem;
  font-size: 1.4rem;
}

.c10 {
  display: none;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin: 8rem auto 0;
  max-width: 100%;
}

.c24 {
  max-width: 100%;
}

.c36 {
  width: 30%;
  margin: auto;
}

.c25 {
  background-color: #fafafa;
  padding: 7rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 2rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c34 {
  color: #4F4C4C;
  font-size: 2.0rem;
}

@media (max-width:768px) {
  .c38 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (min-width:1920px) {
  .c39 {
    font-size: 2.8rem;
  }
}

@media (max-width:768px) {
  .c39 {
    width: 90%;
  }
}

@media (min-width:1920px) {
  .c40 {
    font-size: 1.4rem;
  }
}

@media (max-width:768px) {
  .c40 {
    margin: 1rem auto;
    width: 80%;
  }
}

@media (max-width:1024px) {
  .c11 {
    height: 100%;
  }
}

@media (max-width:1024px) {
  .c14 {
    height: 100%;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-box-pack: space-around;
    -webkit-justify-content: space-around;
    -ms-flex-pack: space-around;
    justify-content: space-around;
  }
}

@media (max-width:1024px) {
  .c1 .c4 {
    width: 6rem;
    position: fixed;
    top: 1rem;
    left: 1rem;
  }

  .c1 .c9 {
    position: fixed;
    top: 1.4rem;
    right: 1rem;
    display: block;
    width: 4rem;
    cursor: pointer;
  }

  .c1 .c16 {
    visibility: 0;
    overflow: hide;
    height: 0;
    opacity: 0;
    pointer-events: none;
    position: absolute;
    top: 9rem;
    -webkit-transition: all 0.2s ease-in-out;
    transition: all 0.2s ease-in-out;
  }

  .c1 .c19 {
    visibility: 0;
    overflow: hide;
    height: 0;
    opacity: 0;
    pointer-events: none;
    -webkit-transition: all 0.5s ease-in-out;
    transition: all 0.5s ease-in-out;
  }

  .c1 .c22 {
    visibility: 0;
    overflow: hide;
    height: 0;
    opacity: 0;
    pointer-events: none;
    -webkit-transition: all 0.5s ease-in-out;
    transition: all 0.5s ease-in-out;
  }

  .c1 .c12 {
    justify-self: center;
    visibility: 0;
    overflow: hide;
    height: 0;
    opacity: 0;
    pointer-events: none;
    position: absolute;
    top: 16rem;
    -webkit-transition: all 0.2s ease-in-out;
    transition: all 0.2s ease-in-out;
  }

  .c1 .c12 ul,
  .c1 .c12 li {
    pointer-events: none;
  }
}

@media (min-width:1920px) {
  .c2 {
    width: 70%;
    margin: auto;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (min-width:1366px) {
  .c5 {
    width: 9rem;
  }
}

<div
  class="c0"
>
  <header
    class="c1"
  >
    <div
      class="c2"
    >
      <div
        class="c3 c4 c5"
        data-testid="Wrapper"
      >
        <span
          class="c6"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjAwIiBoZWlnaHQ9IjUxMiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Logo Lorem Ipsum"
              class="c7"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </div>
        </span>
      </div>
      <svg
        aria-hidden="true"
        class="c8 c9 c10"
        data-testid="MenuIcon"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M88 152h336M88 256h336M88 360h336"
          fill="none"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-miterlimit="10"
          stroke-width="48"
        />
      </svg>
      <nav
        class="c11 c12 c13"
      >
        <ul
          class="c14"
        >
          <li
            class="c15"
          >
            Link 1
          </li>
          <li
            class="c15"
          >
            Link 2
          </li>
          <li
            class="c15"
          >
            Link 3
          </li>
          <li
            class="c15"
          >
            Link 4
          </li>
          <li
            class="c15"
          >
            Link 5
          </li>
          <li
            class="c15"
          >
            Link 6
          </li>
        </ul>
      </nav>
      <div
        class="c16 c17"
      >
        <button
          class="c18 c19 c20"
        >
          Sign In
        </button>
        <button
          class="c21 c22 c23"
        >
          Sign Up
        </button>
      </div>
    </div>
  </header>
  <div
    class="c24"
  >
    <section
      class="c25"
    >
      <form
        class="c26"
      >
        <div
          class=""
        >
          <label
            class="c27"
            for="email"
          >
            Email
          </label>
          <div
            class="c28"
          >
            <input
              class="c29 "
              id="email"
              name="email"
              placeholder="Your email"
              type="email"
              value=""
            />
          </div>
        </div>
        <div
          class=""
        >
          <label
            class="c27"
            for="password"
          >
            Password
          </label>
          <div
            class="c28"
          >
            <div
              class="c30"
            >
              <svg
                aria-hidden="true"
                class="c8 c31"
                data-testid="eye-slash-icon"
                fill="currentColor"
                focusable="false"
                viewBox="0 0 16 16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M13.359 11.238C15.06 9.72 16 8 16 8s-3-5.5-8-5.5a7.028 7.028 0 0 0-2.79.588l.77.771A5.944 5.944 0 0 1 8 3.5c2.12 0 3.879 1.168 5.168 2.457A13.134 13.134 0 0 1 14.828 8c-.058.087-.122.183-.195.288-.335.48-.83 1.12-1.465 1.755-.165.165-.337.328-.517.486l.708.709z"
                />
                <path
                  d="M11.297 9.176a3.5 3.5 0 0 0-4.474-4.474l.823.823a2.5 2.5 0 0 1 2.829 2.829l.822.822zm-2.943 1.299.822.822a3.5 3.5 0 0 1-4.474-4.474l.823.823a2.5 2.5 0 0 0 2.829 2.829z"
                />
                <path
                  d="M3.35 5.47c-.18.16-.353.322-.518.487A13.134 13.134 0 0 0 1.172 8l.195.288c.335.48.83 1.12 1.465 1.755C4.121 11.332 5.881 12.5 8 12.5c.716 0 1.39-.133 2.02-.36l.77.772A7.029 7.029 0 0 1 8 13.5C3 13.5 0 8 0 8s.939-1.721 2.641-3.238l.708.709zm10.296 8.884-12-12 .708-.708 12 12-.708.708z"
                />
              </svg>
            </div>
            <input
              class="c32 "
              id="password"
              name="password"
              type="password"
              value=""
            />
          </div>
        </div>
        <button
          class="c33 "
          type="submit"
        >
          Sign in
        </button>
      </form>
      <span
        class="c34"
      >
        Or
      </span>
      <button
        class="c35 c36"
      >
        <svg
          aria-hidden="true"
          class="c8 "
          fill="currentColor"
          focusable="false"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            data-name="Layer 2"
          >
            <path
              d="M17.5 14a5.51 5.51 0 01-4.5 3.93 6.15 6.15 0 01-7-5.45A6 6 0 0112 6a6.12 6.12 0 012.27.44.5.5 0 00.64-.21l1.44-2.65a.52.52 0 00-.23-.7A10 10 0 002 12.29 10.12 10.12 0 0011.57 22 10 10 0 0022 12.52v-2a.51.51 0 00-.5-.5h-9a.5.5 0 00-.5.5v3a.5.5 0 00.5.5h5"
              data-name="google"
            />
          </g>
        </svg>
        Login by Google
      </button>
    </section>
  </div>
  <footer
    class="c37"
  >
    <div
      class="c38"
    >
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzUiIGhlaWdodD0iMzUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <img
          alt="ISO 9001-2015"
          class=""
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </div>
      <p
        class="c39"
      >
        Etiam dignissim diam quis enim lobortis scelerisque.
      </p>
    </div>
    <span
      class="c40"
    >
      Id diam maecenas ultricies mi eget. Aenean sed adipiscing diam donec adipiscing.
    </span>
    <span
      class="c40"
    >
      Phasellus faucibus scelerisque eleifend donec pretium vulputate sapien. Imperdiet dui accumsan sit amet nulla facilisi morbi tempus.
    </span>
  </footer>
</div>
`;
